README.md

```diff
@@ -5,23 +5,21 @@
 
 ## What is CNI?
 
-CNI, the _Container Network Interface_, is a project to help configure network interfaces for Linux application containers.
-CNI concerns itself with setting up network connectivity of containers and removing allocated resources when the container is deleted.
+CNI, the _Container Network Interface_, specifies how to write plugins that configure network interfaces in Linux containers.
```

Comment:
> How about "CNI .. is a specification and library for writing plugins that ..."?

That would still neglect the plugins we already maintain upstream. Here's my try: 

"The CNI project consists of a specification and library for writing Linux container network configuration plugins, alongside a number of officially supported plugins."